spark/src/test/scala/geotrellis/spark/io/cassandra/CassandraRasterCatalogSpec.scala

```diff
@@ -67,7 +82,8 @@ class CassandraRasterCatalogSpec extends FunSpec
             val tile = getTile(key)
             (tile.cols, tile.rows) should be ((512, 512))
           }
-          
+          // FIXME: Expected exception geotrellis.spark.io.package$LayerNotFoundError to be thrown,
+          // but java.lang.RuntimeException was thrown. (CassandraRasterCatalogSpec.scala:87
```

Comment:
what's going on with this FIXME? (Also we generally use `TODO` instead of `FIXME`)